src/main/java/world/anhgelus/molehunt/Game.java

```diff
@@ -159,11 +160,7 @@ public List<ServerPlayerEntity> getMoles() {
     }
 
     public String getMolesAsString() {
-        return moles.stream()
-                .map(ServerPlayerEntity::getDisplayName)
-                .filter(Objects::nonNull)
-                .map(Text::getString)
-                .collect(Collectors.joining(", "));
+        return moles.stream().map(ServerPlayerEntity::getDisplayName).filter(Objects::nonNull).map(Text::toString).collect(Collectors.joining(", "));

```

Comment:
I prefer the old style. Revert this.

Reply:
Looks like the IDE just made its things x)